collections/hub/golden_path/feature_tests/backward_compatibility/fspiop_protocol_validation.json

```diff
@@ -66,7 +66,7 @@
           "id": 2,
           "description": "Add Participant to ALS",
           "meta": {
-            "info": "This request allows us to add a new participant to Account Lookup Service."
+            "info": "This request allows us to add a new participant to Account Lookup Service.\n\nValidate Not Support Accept header."
```

Comment:
```suggestion
            "info": "This request allows us to add a new participant to Account Lookup Service.\n\nValidate Not Supported Accept header."
```

Reply:
@elnyry-sam-k those tests are not applicable, I replaced them with a `GET Parties` instead. Please see updates.